core/shared/src/main/scala/zio/Schedule.scala

```diff
@@ -25,22 +25,10 @@ object Schedule {
    */
   final val forever: Schedule[Any, Int] = ZSchedule.forever
 
-  /**
-   * See [[ZSchedule.decision]]
-   */
-  final val decision: Schedule[Any, Boolean] =
-    ZSchedule.decision
-
-  /**
-   * See [[ZSchedule.delay]]
-   */
-  final val delay: Schedule[Any, Duration] =
-    ZSchedule.delay
-
   /**
    * See [[ZSchedule.never]]
    */
-  final val never: Schedule[Any, Nothing] =
+  final val never: Schedule[Any, Unit] =
```

Comment:
`Unit` doesn't have the same composition properties as `Nothing` — the old `never` could be `>>>` into a `Schedule` requiring any particular type.